### PR TITLE
Api/1200/volume attachment

### PR DESCRIPTION
**Pull request by sijeesh**
### Description
Updated volume attachment resource file and tested with the latest API version 800,1000 and 1200

### Check List
- [ ] New functionality includes testing.
  - [x] All tests pass (`$ rake test`).
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

**Review by AnushaY1916 — COMMENTED** [review]
API1200 column missing in endpoint file

**Comment by sijeesh** [comment]
> API1200 column missing in endpoint file

It's not missing, you have to scroll horizontally as the list is big now. We should remove all the old versions from the list when we revisit the library next time.

**Review by madhav-bharadwaj — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -25,6 +25,7 @@
 - Server profile
 - Server profile template
 - Uplink set
+- Volume attachment
```
Comment (jyothisgm) [inline CHANGELOG.md:33]: mark it as storage volume attachment.

**Review by jyothisgm — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (sijeesh) [reply CHANGELOG.md:33]: This resource's provider name is volume_attachment and the previous releases also used the same name (volume attachment)

**Review by AnushaY1916 — APPROVED** [review]

**Review by jyothisgm — APPROVED** [review]
LGTM

**Pull request merged**